shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -824,6 +825,32 @@ public String getDartEntrypointFunctionName() {
     }
   }
 
+  /**
+   * The Dart library uri for the entrypoint that will be executed as soon as the Dart snapshot is
+   * loaded.
+   *
+   * <p>Example value: "package:foo/bar.dart"
+   *
+   * <p>This preference can be controlled by setting a {@code <meta-data>} called {@link
```

Comment:
btw, the manifest can have dynamic values that are passed from the tool to Gradle, then copied into the final manifest. e.g. https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/app_shared/android.tmpl/app/src/main/AndroidManifest.xml.tmpl#L5

cc @GaryQian

Reply:
This needs another migration of existing manifest files